x/gov/endblocker.go

```diff
@@ -69,8 +78,8 @@ func EndBlocker(ctx sdk.Context, keeper Keeper) sdk.Tags {
 
 		logger.Info(
 			fmt.Sprintf(
-				"proposal %d (%s) tallied; passed: %v",
-				activeProposal.ProposalID, activeProposal.GetTitle(), passes,
+				"proposal %d (%s) tallied; passed: %v; handler error: %v",
```

Comment:
more from previous comment... Is this referring to a situation where the proposal passes, however the parameter change implementation causes errors when it affects the blockchain state?

Reply:
Yes, more generally, when the proposal passed according to the governance logic but somehow the actual handling logic failed to mutate the state, for example, the proposal content has an invalid format.

Reply:
thanks